certified-connectors/RescoReports/README.md

```diff
@@ -0,0 +1,35 @@
+## Resco Reports
```

Comment:
@JuroJ Please include all required sections in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

Reply:
Done